firefox-ios/Client/Coordinators/Library/BookmarksCoordinator.swift

```diff
@@ -26,20 +27,21 @@ extension BookmarksCoordinatorDelegate {
     func showBookmarkDetail(
         bookmarkType: BookmarkNodeType,
         parentBookmarkFolder: FxBookmarkNode,
-        updatePanelState: ((LibraryPanelSubState) -> Void)? = nil
+        updatePanelState: ((LibraryPanelSubState) -> Void)? = nil,
+        parentFolderSelector: ParentFolderSelector? = nil
     ) {
         showBookmarkDetail(
             bookmarkType: bookmarkType,
             parentBookmarkFolder: parentBookmarkFolder,
-            updatePanelState: updatePanelState
+            updatePanelState: updatePanelState,
+            parentFolderSelector: parentFolderSelector
         )
     }
 }
 
 class BookmarksCoordinator: BaseCoordinator,
                             BookmarksCoordinatorDelegate,
                             QRCodeNavigationHandler,
-                            BookmarksRefactorFeatureFlagProvider,
```

Comment:
I replaced the protocol conformance with a boolean passed in the init of that coordinator so we can easily test even with the feature flag code. Feature flag is just a boolean in this case, so I treat it as is here